### ISSUE TITLE
[Accessibility][ContextualMenu] Initial focus is set on menu, not first menu item

### ISSUE THREAD

**Issue by daweffe** [OfficeDev/office-ui-fabric-react#4207]
### Bug Report

- __Package version(s)__: 5.49.0
- __Browser and OS versions__: all

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: Normal

Products/sites affected: Planner

#### Describe the issue:

When a ContextualMenu is opened, focus is set to the menu. This is confusing for sighted and screen reader keyboard users. For sighted users there is no indication anything has focus. And for screen reader users, it just announces "menu." 

This also doesn't meet MAS (Microsoft Accessibility Standards) which requires items in tab order to be interactive. The whole menu is not an interactive element.

#### Actual behavior:

Focus is set on the whole menu.

#### Expected behavior:

Focus should be set on the first menu item.

### If applicable, please provide a codepen repro:

The sample page documentation shows the behavior nicely.

**Comment by Jahnp** [comment]
Thanks for reporting this @daweffe. @cschlechty , @betrue-final-final -- I believe we spoke about this recently and I thought the behavior @daweffe describes was by design, though I could be wrong on that. Thoughts?

**Comment by betrue-final-final** [comment]
Per a large conversation on the subject offline, here is what we want to happen. @rebeccaballantyne may be working on this as we speak.

•	Opened via touch/mouse 
o	Focus should go to the whole menu. I believe the ContextualMenu in fabric currently allows for the an aria-label on the whole menu which gives a good experience for what is read out (and the user can query easily about what the menu contains with an AT)
•	Opened via keyboard (or programmatically (via AT)) 
o	Focus should go to the first item, it will allow the user to already be placed in the first item, can get info about the list (item 1 of x) and potentially the menu label (depending on the verbosity/contextual announcements of the AT)

**Comment by rebeccaballantyne** [comment]
I've got a PR out for this right now.

**Comment by micahgodbolt** [comment]
@rebeccaballantyne does your PR resolve this issue?

**Comment by rebeccaballantyne** [comment]
Yes.
